ui/src/main/less/bootstrap-3.2.0/variables.less

```
@@ -240,7 +240,7 @@
 @dropdown-caret-color:           #000;
 
 
-//-- Z-index master list
+//-- Z-index controller list
```

This is the authoritative list.  Not a reference to the Jenkins controller.  Either "authoritative" or "primary" could be used.
```suggestion
//-- Z-index primary list
```